packages/redux-query-react/index.d.ts

```diff
@@ -22,7 +22,10 @@ declare module 'redux-query-react' {
   }
   export type ForceRequestCallback<TEntities = Entities> = () => Promise<
     ActionPromiseValue<TEntities>
-  >;
+  > | void;
+
+  export type ForceRequestsCallback = () => void;
```

Comment:
technically its typed the same as `useRequest`

https://github.com/amplitude/redux-query/blob/master/packages/redux-query-react/src/hooks/use-requests.js#L49-L51

but `forceRequest` doesn't return anything:

https://github.com/amplitude/redux-query/blob/master/packages/redux-query-react/src/hooks/use-requests.js#L111-L118